themes/ddbasic/sass/components/view/view.scss

```diff
@@ -66,11 +66,13 @@
     // Styleguide 2.1.9
 
     // Small grid
+    @at-root .grid-small#{&},
```

Comment:
With @at-root it's possible to add a class to the top parent of your sass hierarchy. We use it here to add the grid-classes to our top class which is ".view". 

.view {
  @at-root .grid-small#{&} {
  }
}

Compiles to
.grid-small.view {
}

The sass is still dependent on Views markup, but the grid-classes "grid-small", "grid-medium" and "grid-large" can now be placed on both the panel containing the view and the view itself.